docs/model_pipeline.rst

```diff
@@ -93,9 +105,35 @@ Time series
    :file: ../oemof_b3/schema/timeseries.csv
 
 
-A few more conventions are important to know. Missing data is left empty. If a value applies to all
-regions, this is indicated by :attr:`ALL`. If it applies to the sum of regions, by :attr:`TOTAL`.
-There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
+.. _conventions_resources_label_:
+
+Conventions
+^^^^^^^^^^^
+
+A few more conventions are important to know:
+
+* Missing data is left empty.
+* There is no unit transformation within the model, i.e. the user needs to ensure the consistency of units.
+
+* The parameters :attr:`id_scal` and :attr:`id_ts` are optional and will be added automatically if you do not specify them.
+
+* The parameter :attr:`scenario_key` allows you to pass values that are associated with a specific scenario.
```

Comment:
This information should be given in the schema/*.csv

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/280/commits/84b62fce7afc0ddd6b756d8a2d903990badc0f7c.